### PR TITLE
Details popup window

### PR DESCRIPTION
**Pull request by vanelnw**
in this project we:

- Refactor our main code to add work section dynamically using javascript
- Add popup and styled it.
- use javascript to appear popup when click on each project and disappear when click on cancel button

### CODE REVIEW

**Review by Danieldotcomcoder — COMMENTED** [review]
Hi @vanelnw , 


While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that doesn't include a professional Readme File , Kindly refer to this [template](https://github.com/microverseinc/readme-template/blob/master/README.md) 👍 👍  to add a good Readme File 😄 😄 then submit again, Thank you 👍 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Invalid Code Review Request ❌

Hi @vanelnw, This is @sja-thedude your code reviewer.

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

You have submitted a project, that doesn't have commits from both partners in your pair programming group. Make sure both contributes to the repo.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Comment by vanelnw** [comment]
hi @sja-thedude , thank you for your feedback.  
there is commit of my coding partner in the project.
the one named "Add work popup functionality"  was made by my partner.

**Comment by sja-thedude** [comment]
> hi @sja-thedude , thank you for your feedback.  
> 
> there is commit of my coding partner in the project.
> 
> the one named "Add work popup functionality"  was made by my partner.

Ohh okay maybe I missed it. You could again ask for review.

File: style.css [hunk 1]
```diff
@@ -397,6 +397,90 @@ button:hover {
...
+  transition: 0.5s;
+}
+
+.open {
+  opacity: 1;
+  visibility: visible;
+}
+
+.work-popup-content {
+  position: relative;
```
Comment (BertrandConxy) [inline style.css:422]: - [ ] Good job so far 👏 Your popup window works fine and the popup window is aligned well. Only a little fall out occurs on some project cards where the popup window buttons somehow fall out of view. Take a look at the pic below 👇 
image

TIP: I think you may consider specific height to the image in viewport units or percentages

**Review by BertrandConxy — CHANGES_REQUESTED** [review]
Hi @vanelnw, 

Good job so far! 👏 👏 
There are some issues that you still need to work on to go to the next project but you are almost there! 🏃‍♂️ 🏃‍♂️ 


## Highlights 💪 
- No linter errors
- correct flow is used
- good PR

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @BertrandConxy in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by victorokoroji — APPROVED** [review]
Hi @vanelnw, 

# ✔️   Status: APPROVED 🏆 🎉 🎊 

You have done a great job on this project, keep it up 👍 🥇. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
congratulations!  🎉💯🌟 

## Top highlight 🎯

- The Pull Request is following the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅
- Linters are working properly ✅



## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**